BubbleSort/BubbleSort/BubbleSort/Sort.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Sort;
+
+using System;
+using System.Collections.Generic;
+
+public class Sort
```

Comment:
```suggestion
public static class Sort
```
Ну и конечно, нужны комментарии